content/docs/fragments.md

```diff
@@ -3,8 +3,7 @@ id: fragments
 title: Fragments
```

Comment:
Hey @RamirezAlex, just to be clear, we want to translate the `Title` but not the `permalink`. That said, let's rename this to `Fragmentos`, and start using `Fragmentos` instead of `Fragments` in this document.

Thanks again!